app/javascript/app/pages/home/user-cases-section/user-cases-section-component.jsx

```diff
@@ -51,13 +51,22 @@ class UserCasesSectionComponent extends Component {
               fade
             >
               <div>
-                <img src={politicianImg} alt="Political building" />
+                <div
```

Comment:
Maybe loop here will be a good idea :)
Do we need this <div> parent for each child?

Reply:
Sorry 🤦‍♂️ 
Yeah the parent is needed because of the way the slider works under the hood.

Reply:
I thought so, thanks :)